src/Drupal/V8/BasicCommands.php

```diff
@@ -112,24 +122,49 @@ public function make($opts = [
 
         $this->mergeComposerConfig();
 
-        $this->_exec("composer --no-progress --working-dir={$docroot} {$boolOptions}--prefer-{$opts['prefer']} update");
-        $this->linkSitesDefault();
-        $this->linkModules();
-        $this->linkThemes();
-        $this->linkLibraries();
+        $this->_exec("composer --no-progress --working-dir={$docroot} {$boolOptions} install");
```

Comment:
@dafeder We need to use update instead of install here. Install will use the information in the lock file which does not know anything about the modification that we made to composer.json, so effectively we do not get dkan or its dependencies.

Reply:
Fixed in latest commit: https://github.com/GetDKAN/dkan-tools/pull/131/commits/6ed96bdf942e51496fb8091bc07ed9b314988331